src/python/grpcio/grpc/__init__.py

```diff
@@ -1273,7 +1273,7 @@ def secure_channel(target, credentials, options=None):
                             credentials._credentials)
 
 
-def server(thread_pool, handlers=None, options=None):
+def server(thread_pool, handlers=None, options=None, max_requests=None):
```

Comment:
If you don't care too much about this, [this is something about which I do have a pretty strong desire](https://www.youtube.com/watch?v=aOEfIrC07XA&t=26m25s).

Looking only the whole of the public API, I think we only have two abbreviations: "fn", because we're deliberately maintaining as-much-as-possible compatibility with another API, and "Auth", because... well I didn't think to object at the time (and I'm not sure I would have).

Internally `max_` is fine but here in the public API I'd much appreciate if we can spell out `maximum_rpc_concurrency`. Or `maximum_concurrent_rpcs`?